packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -119,7 +119,7 @@ module.exports = ({ strapi }) => {
     async register() {
       extendReviewWorkflowContentTypes({ strapi });
       strapi.hook('strapi::content-types.afterSync').register(enableReviewWorkflow({ strapi }));
-      strapi.hook('strapi::content-types.afterSync').register(persistStagesJoinTables({ strapi }));
+      strapi.hook('strapi::content-types.beforeSync').register(persistStagesJoinTables({ strapi }));
```

Comment:
why did you changed the order here ? 🤔

Reply:
I think it's safer if we persist or unpersist tables before the sync, for this precise fix is useful, as it will change the persistedTable format to be valid.

Reply:
I see this actually doesnt work on fresh starts of strapi, where the store table has not been creataed. I am reverting this.